tests/e2e/test_building.py

```diff
@@ -7,3 +9,68 @@ def test_default_build(protostar):
     protostar(["build"])
     dirs = listdir()
     assert "build" in dirs
+
+
+@pytest.mark.usefixtures("init")
+def test_output_dir(protostar):
+    protostar(["build", "--output", "out"])
+    dirs = listdir()
+    assert "build" not in dirs
+    assert "out" in dirs
+
+
+@pytest.mark.usefixtures("init")
+def test_cairo_path_argument(protostar, tmpdir, copy_fixture):
+    my_private_libs_dir = Path(tmpdir) / "my_private_libs"
+    mkdir(my_private_libs_dir)
+
+    my_lib_dir = my_private_libs_dir / "my_lib"
+    mkdir(my_lib_dir)
+
+    copy_fixture("simple_function.cairo", my_lib_dir / "utils.cairo")
+
+    with open("./src/main.cairo", mode="w", encoding="utf-8") as my_contract:
+        my_contract.write(
+            """%lang starknet
+from my_lib.utils import get_my_number
+
+@view
+func my_func() -> (res: felt):
+    let (res) = get_my_number()
+    return (res)
+end
+"""
+        )
+
+    protostar(["build", "--cairo-path", my_private_libs_dir])
+
+    dirs = listdir()
+    assert "build" in dirs
+
+
+@pytest.mark.usefixtures("init")
+def test_disable_hint_validation(protostar):
+
+    with open("./src/main.cairo", mode="w", encoding="utf-8") as my_contract:
```

Comment:
You can use `copy_fixture` fixture here, and cairo files, would be a bit cleaner